app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<h2>Sign up</h2>
+
+<div>
+  <div>
+       <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Username' %>
+  </div>
+
+    <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true, autocomplete: "photo", placeholder: 'URL' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: 'Password' %>
+  </div>
+
+  <div>
+    <%= f.submit "Sign up" %>
+  </div>
+
+    <%= render "devise/shared/links" %>
+<% end %>
+  </div>
+
+</div>
```

Comment:
- [ ] Kindly redesign the sign up form also.

  ![reg-ric](https://github.com/RicardoEgas/blog-app/assets/54848363/e5e42249-434b-4bb0-8cd9-35ff5eab5c57)